tests/integration/monitoring/test-monitoring.go

```diff
@@ -31,15 +31,21 @@ const expectedGrafanaInstance = 1
 
 var kubeConfig *rest.Config
 var k8sClient *kubernetes.Clientset
+var monitoringClient *monitoringv1.MonitoringV1Client
 var httpClient *http.Client
 
+// void struct is used to simulate a set data structure using a map
+type void struct{}
```

Comment:
Is there a reason to have this type? There is no need for a type to represent/declare an empty struct.

Reply:
I wanted to have a `set` to store scrape pools inside it, so I'm using a `map` to imitate the behavior of a `set` similar to the 2nd idea here https://yourbasic.org/golang/implement-set/

I'm using the `void` struct here https://github.com/kyma-project/kyma/pull/9481/files#diff-94952d952328e43b83c847ec008d2744R271 and I should have used it in the declaration here as well https://github.com/kyma-project/kyma/pull/9481/files#diff-94952d952328e43b83c847ec008d2744R260, but I missed that one out.

Do you think it would be better to remove the `void` type and replace `scrapePools[scrapePool] = void{}` with `scrapePools[scrapePool] = struct{}{}`?

Reply:
Yes, it would be better to replace `void` with `struct{}{}`. The first one adds unnecessary complexity and indirection, whereas the second is standard and explicit.